main/civ_report_analyzer.py

```diff
@@ -23,6 +23,26 @@
 spaced_indentation = ' ' * 8
 
 
+def get_failed_tests_analysis_2(data):
```

Comment:
Is this function left from debugging? It seems to be a duplicate

Reply:
Good catch! My mistake. I'll remove it!